plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -25,6 +25,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/pkg/api/resource"
 	"k8s.io/kubernetes/pkg/api/v1"
+	metav1 "k8s.io/kubernetes/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/client/cache"
```

Comment:
I'm not a huge fan of introducing a dep on the api-cache to improve coverage.  Currently there are ongoing efforts to eliminate unnecessary dependencies in the code such that algorithm could be lifted to be used externally.  I preferred the FakeNodeList tbh.